cookbook/direnv.md

```diff
@@ -18,8 +18,13 @@ $env.config = {
   hooks: {
     pre_prompt: [{ ||
       let direnv = (direnv export json | from json)
-      let direnv = if ($direnv | length) == 1 { $direnv } else { {} }
-      $direnv | load-env
+      let no_changes = $direnv | is-empty
+      let changes = not $no_changes
+
+      if $changes {
```

Comment:
```suggestion
      if not ($direnv | is-empty) {
```
I think the above will work. Then I'd just delete the no_changes and changes lines.

Reply:
sounds simple enough to me that it does not require two intermediate variables, agree with @fdncred :+1: